Dockerfile.keyserver

```diff
@@ -12,6 +12,7 @@ LABEL org.label-schema.build-date=$BUILD_DATE
 LABEL org.label-schema.vcs-url="https://github.com/EGA-archive/ega-data-api"
 LABEL org.label-schema.vcs-ref=$SOURCE_COMMIT
 COPY --from=builder ega-data-api-key/target/ega-data-api-key-0.0.1-SNAPSHOT.jar .
+COPY ega-data-api-key/src/test/resources .
```

Comment:
Don't copy test resources into the final container.

Reply:
I second that above :arrow_up:

Reply:
Done